deploy/kubernetes/fluentd-sumologic.yaml

```diff
@@ -155,7 +155,7 @@ spec:
   selector:
     matchLabels:
       k8s-app: fluentd-sumologic
-  replicas: 1
+  replicas: 3
```

Comment:
Is there any data duplication with 3 replicas of Fluentd running? Or when Prometheus writes to the remote write endpoint do the metrics only get sent to one of the 3 replicas?

Reply:
No data duplication. the idea is much similar as our receiver nodes. from outside (e.g. Prometheus) there is only single endpoint (which we defined as a kubernetes service). The traffic will be distributed to 3 nodes. We don't have any session or partition concept here so basically it's just randomly pick anyone

Reply:
Nice, thanks for confirming